content/en/blog/_posts/2025-XX-XX-endpoints-deprecation.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: blog
+title: "Endpoints formally deprecated in favor of EndpointSlices"
```

Comment:
With this title, we can only publish it on the day that we formally announce / confirm deprecation.

Maybe we want to have two articles:
1. how to adopt EndpointSlice and stop using Endpoints
1. Endpoints is now formally deprecated, go read the other article to find out what to do

?

Reply:
changed the title to be less authoritative-sounding